app/views/categories/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<% if notice %>
+  <p class="alert alert-success"><%= notice %></p>
+<% end %>
+<% if alert %>
+  <p class="alert alert-danger"><% alert %></p>
+<% end %>
+<section class="container>
+  <%= link_to category_path(@category.id) do %>
+    <article class="row border my-1">
+      <div class="col-3">
+        <img src=<%= @category.icon %> alt="Category Icon" class="m-1 img-fluid rounded-1" > 
+      </div>
+      <div class="col-6 pt-3">
+        <p class="category-name"><%= @category.name %></p>
+        <span class="date" ><%= @category.created_at %> </span>
+      </div>
+      <div>
+        <span class="amount-dsl">$<%= @category.amount %></span>
+      </div>
+    </article>
+  <% end %>
+  <br>
+  <% @transacs.each do |trans| %>
+    <article class="row border my-1">
+      <div class="col-9">
+        <p class="trans-name">Transac <%= trans.name %></p>
+        <p class="date"><%= trans.created_at  %><%= trans.name %></p>
+      </div>
+       <div>
+        <span class="amount">$<%= trans.amount %><span>
+      </div>
+    </article>
+  <% end %>
+</section>
+
+<div class="bottom-container position-fixed bottom-o w-100" >
+  <div class="inner-bottom-container text-center">
+    <%= link_to "NEW TRansac", new_transac_path, class:"buttom-btn" %>
+  </div>
+</div>
```

Comment:
- [ ] After clicking on the category ı should see transactions that belong to the category. You made another page for transactions and ı can see all transactions in there. Kİndly fix it
![image](https://user-images.githubusercontent.com/38635158/184506674-7751428a-d9d3-4b5f-a9c2-42ed58ab84b1.png)